apps/src/javalab/CommitDialog.jsx

```diff
@@ -41,24 +40,57 @@ export default class CommitDialog extends React.Component {
       >
         {compileStatusContent}
       </div>,
-      <div key="buttons">{buttons}</div>
+      <div key="buttons">
+        <FooterButton
+          key="cancel"
+          type="cancel"
+          text={i18n.cancel()}
+          onClick={this.props.handleClose}
+        />
+        ,
+        <FooterButton
+          id="confirmationButton"
+          key="confirm"
+          text={commitText}
+          disabled={isCommitButtonDisabled}
+          color="green"
+          onClick={() => {
+            this.props.handleCommit(
```

Comment:
let's move this to a helper function instead of an anonymous function as we will add more logic here once we add the compile step

Reply:
done- thanks🙌